drivers/sensor/lsm6dsl/Kconfig

```diff
@@ -8,24 +8,41 @@
 #
 
 menuconfig LSM6DSL
-	bool "LSM6DSL I2C accelerometer and gyroscope Chip"
-	depends on I2C
+	bool "LSM6DSL I2C/SPI accelerometer and gyroscope Chip"
+	depends on SENSOR && (I2C || SPI)
 	default n
 	help
-	  Enable driver for LSM6DSL I2C-based accelerometer and gyroscope
+	  Enable driver for LSM6DSL accelerometer and gyroscope
 	  sensor.
 
-if !HAS_DTS_I2C_DEVICE
-
 config LSM6DSL_DEV_NAME
```

Comment:
So, actually this one should be under !HAS_DTS_I2C_DEVICE.
Can be put in depend on maybe